docs/resources/conventions.rst

```diff
@@ -88,164 +88,164 @@ Current
      - Comment
    * - GLOBAL_Conventions (*attribute*)
      - SOFA
-     - 
+     -
```

Comment:
this file should be deleted on git since its always recreated if the doc os build. We sould then also add this file to the gitignore.